core/domain/services/graphql/data/mutations/PriceMutation.php

```diff
@@ -23,11 +23,12 @@ public static function prepareFields(array $input)
     {
         $args = [];
 
-        if (! empty($input['amount'])) {
+        // amount can be 0
+        if (array_key_exists('amount', $input)) {
```

Comment:
doh! right of course... I should have caught that during a previous review because every EE dev gets bit by that at some point